docker-compose/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:xenial
+
+ADD geth /root/
+ADD libhera.so /root/
```

Comment:
Where do these files come from?

Reply:
you copy them to this folder.  it's explained in the README.md attached to this PR.

Reply:
They could be built in this docker image...

Reply:
At the bare minimum please include the build of those two in `build.sh`. Don't fully understand why some of it is built by the script, but these two aren't.